CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing to Zonemaster::Backend
+
+Contribution to this repository is welcome. Contribution can be either an issue
+report or a code or a documentation update. Also see the information in the
+[main README][Zonemaster/Zonemaster README] in the main Zonemaster respository.
+
+## Issue
+
+First search for a similar issue in the [issues list]. If a relevant issue is
+found, add your information as a comment. If no relevant issue is found, create a
+[new][create issue] issue. Give as many details as you have and describe, if
```

Comment:
Why not make the link text be "create a new issue"? That way it's clearer where the link is pointing to.

Same goes for the create a new pull request link.

Reply:
Because I dislike long link texts since they stick out too much. Have done a compromise that I will update with.